src/LtiRegistration.php

```diff
@@ -6,16 +6,16 @@
 
 class LtiRegistration implements ILtiRegistration
 {
-    private $issuer;
-    private $clientId;
-    private $keySetUrl;
-    private $authTokenUrl;
-    private $authLoginUrl;
-    private $authServer;
+    private ?string $issuer;
+    private ?string $clientId;
+    private ?string $keySetUrl;
+    private ?string $authTokenUrl;
+    private ?string $authLoginUrl;
+    private ?string $authServer;
     private $toolPrivateKey;
```

Comment:
Was this left untyped because it may either be a `string` or `int`?

Reply:
Ohhh. Cos I had no idea what the type should be and wanted to go re-read the spec to figure it out. Lemme do that.